src/rpc/http-methods/reset-network.md

```diff
@@ -0,0 +1,30 @@
+# `resetNetwork`
+
+**Description:** Resets the network state.
```

Comment:
We should probably provide more explanation on what this does and the situations under which this might be called. It's a pretty dramatic call.

Reply:
Perhaps I should remove it for the time being so there isn't any confusion. I checked the code and we do have an access control check to make sure that the caller is leader. I'll make a note all the same but will defer to you on whether we include this or not.